src/cmake-tools.ts

```diff
@@ -526,7 +592,11 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
    * Implementation of `cmake.stop`
    */
   async stop(): Promise<boolean> {
-    const drv = await this._cmakeDriver;
+    const drv = await this.getCMakeDriverInstance();
+    if (!drv) {
+      return false;
```

Comment:
I think this should be returned as a `Promise`:
```ts
return Promise.resolve(false);
```

Reply:
Style guide question